internals/overlord/servstate/manager_test.go

```diff
@@ -506,14 +377,13 @@ func (s *S) TestStartBadCommand(c *C) {
 }
 
 func (s *S) TestCurrentUserGroup(c *C) {
-	// Don't re-use s.manager, because we're adding a layer and service, and
-	// that would conflict with other tests like TestServiceLogs.
+	// This test supplies its own service manager instance and
+	// therefore should not call the default setup.
```

Comment:
This comment (also below) is probably unnecessary now. All tests supply their own service manager -- some via `defaultServiceManagerSetup` and some inline.